rhoc/pkg/cmd/deployments/snapshot/snapshot.go

```diff
@@ -0,0 +1,176 @@
+package snapshot
+
+import (
+	"errors"
+	"fmt"
+	"github.com/bf2fc6cc711aee1a0c2a/cos-tools/rhoc/pkg/service"
+	"github.com/bf2fc6cc711aee1a0c2a/cos-tools/rhoc/pkg/util/cmdutil"
+	"github.com/bf2fc6cc711aee1a0c2a/cos-tools/rhoc/pkg/util/dumper"
+	"github.com/bf2fc6cc711aee1a0c2a/cos-tools/rhoc/pkg/util/request"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+const (
+	CommandName  = "snapshot"
+	CommandAlias = "ss"
+)
```

Comment:
Why declaring a constant for this package that is only used once, we could as well  directly do:
```
	cmd := &cobra.Command{
		Short:   "Takes a snapshot of deployments in oldest and ready clusters.",
		Use:      "snapshot",
```
In case there is a valid reason, why make it public when it is never accessed from outside the package.

Reply:
I could change it, but that pattern is followed by basically all of the commands.
I don't know what is the reason behind that pattern being used. @lburgazzoli? 

If there is no reason behind it, I can change it if we don't mind it being the different one.

Reply:
we took this commands originally from rhoas and that was the pattern as I recall it was used also outside the package. 

we can do whatever make sense for us.

Reply:
The only point I can see having it this way is that you get a good idea bout what the package is doing by looking at the top of the file, but even for that there are better ways. 

I am just challenging some ideas I know we have been dragging from kas, no strong position whatsoever.